webpack.config.dev.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+const HTMLLoader = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.tsx',
+  mode: 'development',
+  output: {
+    filename: "[name].js",
+    path: path.resolve(__dirname, 'build'),
+  },
+  module: {
+    rules: [
```

Comment:
Highly likely in your react project you will have css, please also tune up webpack for this

Reply:
Seems like yes, but i suppose I can use chakra without plain css, just .t(j)s, I'll remove it at the end if I don't use it.